crates/net/downloaders/src/bodies/bodies.rs

```diff
@@ -249,7 +249,7 @@ where
 impl<B, DB> BodiesDownloader<B, DB>
 where
     B: BodiesClient + 'static,
-    DB: Database,
+    DB: Database + Unpin + 'static,
```

Comment:
is this static bound OK for library use?

Reply:
yeh, it's just a type bound